touch/resistive/fourwire.go

```diff
@@ -0,0 +1,82 @@
+package resistive
+
+import (
+	"machine"
+
+	"tinygo.org/x/drivers/touch"
+)
+
+type FourWireTouchscreen struct {
+	YP machine.Pin
+	YM machine.Pin
+	XP machine.Pin
+	XM machine.Pin
+
+	yp machine.ADC
+	xm machine.ADC
+	xp machine.ADC
```

Comment:
It looks like these are essentially duplicates of the pins declared above. It should be possible (and more efficient) to declare them inline:

```
machine.ADC{res.XP}.Get()
```

Reply:
Ok yes, I think that would be clearer.  I wasn't sure if creating a new instance of the struct for each call to Get() would have a negative impact on anything, so I went with just keeping a copy of it around.  I will change it over to the way you've suggested

Reply:
This is addressed in the latest set of commits.